src/components/display/fewlines/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "styled-components";
+
+type RadioButtonProps = {
+  name: string;
+  checked: boolean;
+};
+
+export const RadioButton: React.FC<RadioButtonProps> = ({ name, checked }) => {
+  return (
+    <Wrapper>
+      <label className="radio-container">
+        <input type="radio" name={name} defaultChecked={checked} />
+        <span className="circle"></span>
+      </label>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  .radio-container {
+    position: relative;
+    cursor: pointer;
+    user-select: none;
+    padding-left: 3rem;
+  }
+
```

Comment:
You should remove the `className` and target the html tag right away.
```suggestion
export const RadioButton: React.FC<RadioButtonProps> = ({ name, checked }) => {
  return (
    <Wrapper>
      <label>
        <input type="radio" name={name} defaultChecked={checked} />
        <span/>
      </label>
    </Wrapper>
  );
};

const Wrapper = styled.div`
	label {
		...
	}
	
	span {
		...
	}

```

Reply:
OK fixed 0bf73a0